check-peps.py

```diff
@@ -311,7 +313,7 @@ def _validate_discussions_to(line_num: int, line: str) -> MessageIterator:
             if re.fullmatch(r"[\w\-]+", remainder) is None:
                 yield line_num, "Discussions-To must be a valid mailing list"
             return
-    yield line_num, "Discussions-To must be a valid thread URL or mailing list"
+    yield line_num, "Discussions-To must be 'TBA', a valid thread URL or mailing list"
```

Comment:
Perhaps put the placeholder case last?
```suggestion
    yield line_num, "Discussions-To must be a valid thread URL, mailing list, or 'TBA'"
```